traffic_ops/app/lib/API/Tenant.pm

```diff
@@ -121,7 +115,12 @@ sub update {
 		}	
 	}	
 
-	if ( !defined( $params->{parentId}) && !$self->isRootTenant($id) ) {
+	my $tenant_utils = UI::TenantUtils->new($self);
+	my $tenants_data = $tenant_utils->create_tenants_data_from_db(undef);
+
```

Comment:
yes, i would say the root tenant can't be changed in any way. you can't change it's name, or it's active flag or it's parent (which has to be null). that is a record in the database that can't be changed at all thru the api.